### PR TITLE
[statistics] fix imaging tab loading when Scan_Done is not in mri_parameter_form

### PR DESCRIPTION
**Pull request by ridz1208**
### Brief summary of changes
The imaging tab of statistics depends on the scan types defined in the `mri_scan_type` table. each scan type is crossmatched with a column in the `mri_parameter_form` table. if that column does not exist the SQL query is now bypassed and an error message is displayed stating that the column does not exist

> to review use the hide white spaces changes option

### This resolves issue...

- [ ] Github? #4545

### To test this change...

- [ ] make sure the imaging tab of statistics loads even if the mri_parameter_form does not have all the necessary columns

### CODE REVIEW

File: modules/statistics/php/stats_mri.class.inc [hunk 1]
```diff
@@ -334,18 +341,6 @@ class Stats_MRI extends \NDB_Form
         } else {
             $MRI_Type = $scan_types[$_REQUEST['mri_type']];
         }
-        //TODO IBIS
```
Comment (ridz1208) [inline modules/statistics/php/stats_mri.class.inc]: @sruthymathew123 you should know about these for IBIS

Comment (sruthymathew123) [reply modules/statistics/php/stats_mri.class.inc]: @ridz1208 Thanks Rida. But for some reason IBIS is no more using imaging statistics.

File: modules/statistics/php/stats_mri.class.inc [hunk 2]
```diff
@@ -70,6 +70,7 @@ class Stats_MRI extends \NDB_Form
      * @param string  $Subsection        the value of Subsection=''
      * @param string  $disclamer         the value of disclamer=''
      * @param ?string $projectID         the value of projectID is null
+     * @param bool    $scanDoneExists    does the column exist in the databse
```
Comment (johnsaigle) [inline modules/statistics/php/stats_mri.class.inc]:
```suggestion
     * @param bool    $scanDoneExists    whether the "Scan done" column exists in the database
```

**Comment by johnsaigle** [comment]
Will approve after the above documentation change

**Comment by ridz1208** [comment]
@johnsaigle done

**Comment by ridz1208** [comment]
@johnsaigle approve meeee

**Pull request merged**